_content/articles/divinity-of-the-dalai-lama-scriptural-sources_maccormack-ian.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "The Divinity of the Dalai Lama and its Scriptural Sources: A Study in Tibetan Political Theology"
+authors:
+  - "Ian MacCormack"
+external_url: "https://poj.peeters-leuven.be/content.php?id=3289053&url=article&download=yes"
+drive_links:
+  - "https://drive.google.com/file/d/1r7ztbuS61ZrLhWBw966VSEVB7dSo8c0q/view?usp=drivesdk"
+course: tibetan-roots
+tags:
+  - tibet
+  - political-ideology
+  - society
+year: 2020
+journal: jiabs
+volume: 43
+pages: "81--127"
+---
+
+> The fifth Dalai Lama was figured as the state’s absolute ruler on the basis of a power in excess of his own person. In technical terms, he was the latest in a rebirth line of Avalokiteśvara, here assuming the novel identity of a “renunciate king” or vow-bound sovereign.
+
+In the first part of his two-part study, Ian MacCormack examines the theological foundations of the Dalai Lama's divine kingship as articulated by the Desi Sangyé Gyatso (1653–1705), the 5th Dalai Lama's regent. This nuanced depiction underscores the complex interplay between ultimate truth and conventional reality in Tibetan political theology.
+
+Part two of this study can be found [here](/content/articles/mortality-of-the-dalai-lama_maccormack-ian.md)
```

Comment:
`/content/articles/mortality-of-the-dalai-lama_maccormack-ian`  No `.md` once the software converts it to a webpage 😊